pkg/route/api/v1/types.go

```diff
@@ -82,6 +82,9 @@ type RouteSpec struct {
 
 	// The tls field provides the ability to configure certificates and termination for the route.
 	TLS *TLSConfig `json:"tls,omitempty" protobuf:"bytes,6,opt,name=tls"`
+
+	// Wildcard policy if any for the route.
+	WildcardPolicy WildcardPolicyType `json:"wildcardPolicy,omitempty" protobuf:"bytes,7,opt,name=wildcardPolicy"`
```

Comment:
Thanks did that and it now shows up fine. Took a while to get this all working - missed a protoc version message 3.0.x